test/Middleware/WhoopsErrorResponseGeneratorTest.php

```diff
@@ -155,4 +155,25 @@ public function testJsonContentTypeResponseWithJsonResponseHandler()
             $generator($error, $this->request->reveal(), $this->response->reveal())
         );
     }
+
+    public function testNonErrorStatusCodeIsSetTo500()
+    {
+        $error = new RuntimeException();
+
+        $this->whoops->getHandlers()->willReturn([]);
+        $this->whoops->handleException($error)->willReturn('WHOOPS');
+
+        $this->response->withStatus(StatusCode::STATUS_INTERNAL_SERVER_ERROR)->will([$this->response, 'reveal']);
+        $this->response->getBody()->will([$this->stream, 'reveal']);
+        $this->response->getStatusCode()->willReturn(StatusCode::STATUS_MOVED_PERMANENTLY);
```

Comment:
Uhm, it doesn't make sense since the default error code is 0. So it's already converted and tested in the previous tests.